src/operations_problems_templates.jl

```diff
@@ -197,10 +223,16 @@ results = run_economic_dispatch(system; optimizer = optimizer)
 - `branches::Dict{Symbol, DeviceModel}` : override default `DeviceModel` settings
 - `services::Dict{Symbol, ServiceModel}` : override default `ServiceModel` settings
 - `optimizer::JuMP optimizer` : a JuMP optimizer is a required key word
+- `savepath::AbstractString`  : Path to save results
+- Key word arguments supported by `OperationsProblem`
 """
 
 function run_economic_dispatch(sys::PSY.System; kwargs...)
+    solve_kwargs = Dict()
+    for kw in OPERATIONS_SOLVE_KWARGS
+        haskey(kwargs, kw) && (solve_kwargs[kw] = kwargs[kw])
+    end
```

Comment:
This pattern is an intersection of sets.  In case you want something more clever (and there's probably something even shorter).

```
solve_kwargs = Dict(x => kwargs[x] for x in intersect(keys(kwargs), OPERATIONS_SOLVE_KWARGS))
```